pkg/api/helpers.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 var NameMayNotBe = []string{".", ".."}
-var NameMayNotContain = []string{"/", "%"}
+var NameMayNotContain = []string{"%"}
```

Comment:
Just making sure this doesn't get reintroduced in a future PR. This restriction has to stay in place. The thing we wanted to allow extra slashes in was docker repos, not imagestreams